slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -92,6 +92,52 @@ ::grpc::Status SLAMServiceImpl::GetPositionNew(
     return grpc::Status::OK;
 }
 
+::grpc::Status SLAMServiceImpl::GetPointCloudMap(
+    ServerContext *context, const GetPointCloudMapRequest *request,
+    GetPointCloudMapResponse *response) {
+    std::string pointcloud_map;
+    // Write or grab the latest pointcloud map in form of a string
+    try {
```

Comment:
chose to copy/paste the now-named `GetCurrentPointCloudMap` instead of using that function because the request/response messages are different, and The previous version will no longer be needed once we remove the depreciated endpoints